WireAuthentication/Sources/WireAuthentication/Components/NoHistoryComponent.swift

```diff
@@ -25,6 +25,7 @@ import WireReusableUIComponents
 
 protocol NoHistoryComponentDependency: Dependency {
 
+    var didReauthenticate: Bool { get }
```

Comment:
The "no history" screen now has two modes:
1. "This is the first time you've used this device"
2. "You've used this device before"

To switch between these modes we need to tell it if we have reauthenticated.